packages/kbn-user-profile-components/src/user_profiles_selectable.tsx

```diff
@@ -256,7 +262,12 @@ export const UserProfilesSelectable: FunctionComponent<UserProfilesSelectablePro
           <EuiPanel hasShadow={false} paddingSize="s">
             {search}
             <EuiSpacer size="s" />
-            <EuiFlexGroup alignItems="center" justifyContent="spaceBetween" responsive={false}>
+            <EuiFlexGroup
+              alignItems="center"
+              justifyContent="spaceBetween"
+              responsive={false}
+              gutterSize="none"
```

Comment:
I added `gutterSize none` here to have the component be closer to the mocks, specifically to keep only 8px margin

<img width="570" alt="image" src="https://user-images.githubusercontent.com/56361221/185236093-450f3414-047a-405e-af9a-ff26b3730174.png">

Reply:
The `gutterSize` prop here does not change the padding on the popover and has no visible effect as far as I can see. Are you sure you need this change? If you want to change the popover padding you need to use `panelPaddingSize` on the EuiPopover.  This is something the UserProfilesPopover component already does:

<img width="542" alt="Screenshot 2022-08-18 at 10 39 59" src="https://user-images.githubusercontent.com/190132/185365347-7ac9660f-6d0d-4a1c-93ea-203d0befff1b.png">